testing/py3-spacy/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Maintainer: Oleg Titov <oleg.titov@gmail.com>
+pkgname=py3-spacy
+pkgver=2.2.1
+pkgrel=0
+pkgdesc="Industrial-strength Natural Language Processing (NLP)"
+url="https://spacy.io/"
+arch="x86_64"
```

Comment:
Why is the arch so limited? Please leave a comment

Reply:
Done! Thank you!

Reply:
I have detected that py3-srsly do not passes the test on x86 and aarch64 architectures, reported upstream https://github.com/explosion/srsly/issues/15

Reply:
Thanks (also, I think you meant to say "Alpine Linux", not "Arch Linux" :P)

Reply:
Want to disable tests for now?

Reply:
> Thanks (also, I think you meant to say "Alpine Linux", not "Arch Linux" :P)

Oh, what a shame... Fixed. Thank You!